Python 3.12 breaks BrownieBee stable workflows

Diskutils was deprecated in earlier Python. It seems to be a sub-requirement for something in BB_stable version of ProcessOptimizer. Hence installation of BrownieBee version of Processoptimizer will fail in Python 3.12.
Interesting note from error message is that numpy apprars to be installed as 1.26.4 first, and then is installad as 1.24.4 around the time of the error.

Part of error message:
2024-03-27T09:37:53.4538736Z Requirement already satisfied: pip in /opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages (24.0)
2024-03-27T09:37:54.2889652Z Collecting flake8
2024-03-27T09:37:54.3456364Z   Downloading flake8-7.0.0-py2.py3-none-any.whl.metadata (3.8 kB)
2024-03-27T09:37:54.4423985Z Collecting pytest
2024-03-27T09:37:54.4526181Z   Downloading pytest-8.1.1-py3-none-any.whl.metadata (7.6 kB)
2024-03-27T09:37:54.4821356Z Collecting mccabe<0.8.0,>=0.7.0 (from flake8)
2024-03-27T09:37:54.4925099Z   Downloading mccabe-0.7.0-py2.py3-none-any.whl.metadata (5.0 kB)
2024-03-27T09:37:54.5206566Z Collecting pycodestyle<2.12.0,>=2.11.0 (from flake8)
2024-03-27T09:37:54.5304308Z   Downloading pycodestyle-2.11.1-py2.py3-none-any.whl.metadata (4.5 kB)
2024-03-27T09:37:54.5634425Z Collecting pyflakes<3.3.0,>=3.2.0 (from flake8)
2024-03-27T09:37:54.5731949Z   Downloading pyflakes-3.2.0-py2.py3-none-any.whl.metadata (3.5 kB)
2024-03-27T09:37:54.6061223Z Collecting iniconfig (from pytest)
2024-03-27T09:37:54.6161570Z   Downloading iniconfig-2.0.0-py3-none-any.whl.metadata (2.6 kB)
2024-03-27T09:37:54.6542405Z Collecting packaging (from pytest)
2024-03-27T09:37:54.6640822Z   Downloading packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
2024-03-27T09:37:54.6957562Z Collecting pluggy<2.0,>=1.4 (from pytest)
2024-03-27T09:37:54.7053722Z   Downloading pluggy-1.4.0-py3-none-any.whl.metadata (4.3 kB)
2024-03-27T09:37:54.7293709Z Downloading flake8-7.0.0-py2.py3-none-any.whl (57 kB)
2024-03-27T09:37:54.7425556Z    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.6/57.6 kB 4.7 MB/s eta 0:00:00
2024-03-27T09:37:54.7522755Z Downloading pytest-8.1.1-py3-none-any.whl (337 kB)
2024-03-27T09:37:54.7724460Z    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 337.4/337.4 kB 18.2 MB/s eta 0:00:00
2024-03-27T09:37:54.7820686Z Downloading mccabe-0.7.0-py2.py3-none-any.whl (7.3 kB)
2024-03-27T09:37:54.7939247Z Downloading pluggy-1.4.0-py3-none-any.whl (20 kB)
2024-03-27T09:37:54.8062070Z Downloading pycodestyle-2.11.1-py2.py3-none-any.whl (31 kB)
2024-03-27T09:37:54.8186497Z Downloading pyflakes-3.2.0-py2.py3-none-any.whl (62 kB)
2024-03-27T09:37:54.8251060Z    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.7/62.7 kB 12.9 MB/s eta 0:00:00
2024-03-27T09:37:54.8351554Z Downloading iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
2024-03-27T09:37:54.8472248Z Downloading packaging-24.0-py3-none-any.whl (53 kB)
2024-03-27T09:37:54.8517572Z    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.5/53.5 kB 17.6 MB/s eta 0:00:00
2024-03-27T09:37:54.8876223Z Installing collected packages: pyflakes, pycodestyle, pluggy, packaging, mccabe, iniconfig, pytest, flake8
2024-03-27T09:37:55.3746253Z Successfully installed flake8-7.0.0 iniconfig-2.0.0 mccabe-0.7.0 packaging-24.0 pluggy-1.4.0 pycodestyle-2.11.1 pyflakes-3.2.0 pytest-8.1.1
2024-03-27T09:37:55.8823398Z Obtaining file:///home/runner/work/ProcessOptimizer/ProcessOptimizer
2024-03-27T09:37:55.8840278Z   Installing build dependencies: started
2024-03-27T09:37:57.4646791Z   Installing build dependencies: finished with status 'done'
2024-03-27T09:37:57.4654632Z   Checking if build backend supports build_editable: started
2024-03-27T09:37:57.6618668Z   Checking if build backend supports build_editable: finished with status 'done'
2024-03-27T09:37:57.6626094Z   Getting requirements to build editable: started
2024-03-27T09:37:57.8039798Z   Getting requirements to build editable: finished with status 'done'
2024-03-27T09:37:57.8059345Z   Installing backend dependencies: started
2024-03-27T09:37:58.5821306Z   Installing backend dependencies: finished with status 'done'
2024-03-27T09:37:58.5828092Z   Preparing editable metadata (pyproject.toml): started
2024-03-27T09:37:58.8130973Z   Preparing editable metadata (pyproject.toml): finished with status 'done'
2024-03-27T09:37:59.1264965Z Collecting numpy>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.1816418Z   Downloading numpy-1.26.4-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (61 kB)
2024-03-27T09:37:59.1993397Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.0/61.0 kB 3.4 MB/s eta 0:00:00
2024-03-27T09:37:59.3715368Z Collecting matplotlib>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.3822502Z   Downloading matplotlib-3.8.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.8 kB)
2024-03-27T09:37:59.5165905Z Collecting scipy>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.5272625Z   Downloading scipy-1.12.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
2024-03-27T09:37:59.5358498Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 60.4/60.4 kB 7.9 MB/s eta 0:00:00
2024-03-27T09:37:59.6131837Z Collecting bokeh>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.6240075Z   Downloading bokeh-3.4.0-py3-none-any.whl.metadata (12 kB)
2024-03-27T09:37:59.7332651Z Collecting scikit-learn>=0.24.2 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.7443786Z   Downloading scikit_learn-1.4.1.post1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (11 kB)
2024-03-27T09:37:59.7749317Z Collecting six>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.7855254Z   Downloading six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
2024-03-27T09:37:59.8209885Z Collecting deap>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:37:59.8337757Z   Downloading deap-1.4.1.tar.gz (1.1 MB)
2024-03-27T09:37:59.9265119Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 11.7 MB/s eta 0:00:00
2024-03-27T09:38:00.0030193Z   Installing build dependencies: started
2024-03-27T09:38:01.4332670Z   Installing build dependencies: finished with status 'done'
2024-03-27T09:38:01.4338407Z   Getting requirements to build wheel: started
2024-03-27T09:38:01.5984088Z   Getting requirements to build wheel: finished with status 'done'
2024-03-27T09:38:01.6001595Z   Installing backend dependencies: started
2024-03-27T09:38:02.2817793Z   Installing backend dependencies: finished with status 'done'
2024-03-27T09:38:02.2824797Z   Preparing metadata (pyproject.toml): started
2024-03-27T09:38:02.4670796Z   Preparing metadata (pyproject.toml): finished with status 'done'
2024-03-27T09:38:02.5182600Z Collecting pyYAML>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:38:02.5283397Z   Downloading PyYAML-6.0.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.1 kB)
2024-03-27T09:38:02.5628233Z Collecting bokeh>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:38:02.5738332Z   Downloading bokeh-3.1.1-py3-none-any.whl.metadata (12 kB)
2024-03-27T09:38:02.5890714Z Collecting matplotlib>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:38:02.6019211Z   Downloading matplotlib-3.7.2.tar.gz (38.1 MB)
2024-03-27T09:38:03.2210053Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 38.1/38.1 MB 71.6 MB/s eta 0:00:00
2024-03-27T09:38:04.7630936Z   Installing build dependencies: started
2024-03-27T09:38:09.6018297Z   Installing build dependencies: finished with status 'done'
2024-03-27T09:38:09.6023886Z   Getting requirements to build wheel: started
2024-03-27T09:38:10.5022191Z   Getting requirements to build wheel: finished with status 'done'
2024-03-27T09:38:10.5094766Z   Installing backend dependencies: started
2024-03-27T09:38:11.2119061Z   Installing backend dependencies: finished with status 'done'
2024-03-27T09:38:11.2125386Z   Preparing metadata (pyproject.toml): started
2024-03-27T09:38:11.6567210Z   Preparing metadata (pyproject.toml): finished with status 'done'
2024-03-27T09:38:11.6972735Z Collecting numpy>=1.0.0 (from ProcessOptimizer==0.9.4)
2024-03-27T09:38:11.7111917Z   Downloading numpy-1.24.4.tar.gz (10.9 MB)
2024-03-27T09:38:11.8115519Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.9/10.9 MB 109.5 MB/s eta 0:00:00
2024-03-27T09:38:12.6040325Z   Installing build dependencies: started
2024-03-27T09:38:15.3302272Z   Installing build dependencies: finished with status 'done'
2024-03-27T09:38:15.3332348Z   Getting requirements to build wheel: started
2024-03-27T09:38:15.3864913Z   Getting requirements to build wheel: finished with status 'done'
2024-03-27T09:38:15.4588354Z ERROR: Exception:
2024-03-27T09:38:15.4589303Z Traceback (most recent call last):
2024-03-27T09:38:15.4608244Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/cli/base_command.py", line 180, in exc_logging_wrapper
2024-03-27T09:38:15.4609929Z     status = run_func(*args)
2024-03-27T09:38:15.4610826Z              ^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4612555Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/cli/req_command.py", line 245, in wrapper
2024-03-27T09:38:15.4614208Z     return func(self, options, args)
2024-03-27T09:38:15.4614885Z            ^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4616544Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/commands/install.py", line 377, in run
2024-03-27T09:38:15.4618182Z     requirement_set = resolver.resolve(
2024-03-27T09:38:15.4618940Z                       ^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4620790Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 95, in resolve
2024-03-27T09:38:15.4622567Z     result = self._result = resolver.resolve(
2024-03-27T09:38:15.4623356Z                             ^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4625026Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 546, in resolve
2024-03-27T09:38:15.4626847Z     state = resolution.resolve(requirements, max_rounds=max_rounds)
2024-03-27T09:38:15.4627811Z             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4629545Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 427, in resolve
2024-03-27T09:38:15.4631398Z     failure_causes = self._attempt_to_pin_criterion(name)
2024-03-27T09:38:15.4632234Z                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4634495Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 239, in _attempt_to_pin_criterion
2024-03-27T09:38:15.4636496Z     criteria = self._get_updated_criteria(candidate)
2024-03-27T09:38:15.4637296Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4639278Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 230, in _get_updated_criteria
2024-03-27T09:38:15.4641319Z     self._add_to_criteria(criteria, requirement, parent=candidate)
2024-03-27T09:38:15.4643400Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 173, in _add_to_criteria
2024-03-27T09:38:15.4645074Z     if not criterion.candidates:
2024-03-27T09:38:15.4645854Z            ^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4647539Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/resolvelib/structs.py", line 156, in __bool__
2024-03-27T09:38:15.4649094Z     return bool(self._sequence)
2024-03-27T09:38:15.4650253Z            ^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4652213Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 155, in __bool__
2024-03-27T09:38:15.4653937Z     return any(self)
2024-03-27T09:38:15.4654654Z            ^^^^^^^^^
2024-03-27T09:38:15.4656783Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 143, in <genexpr>
2024-03-27T09:38:15.4658732Z     return (c for c in iterator if id(c) not in self._incompatible_ids)
2024-03-27T09:38:15.4659928Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4662037Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 47, in _iter_built
2024-03-27T09:38:15.4663862Z     candidate = func()
2024-03-27T09:38:15.4664520Z                 ^^^^^^
2024-03-27T09:38:15.4666673Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 182, in _make_candidate_from_link
2024-03-27T09:38:15.4668746Z     base: Optional[BaseCandidate] = self._make_base_candidate_from_link(
2024-03-27T09:38:15.4669732Z                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4672039Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 228, in _make_base_candidate_from_link
2024-03-27T09:38:15.4673684Z     self._link_candidate_cache[link] = LinkCandidate(
2024-03-27T09:38:15.4674229Z                                        ^^^^^^^^^^^^^^
2024-03-27T09:38:15.4675476Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 290, in __init__
2024-03-27T09:38:15.4676508Z     super().__init__(
2024-03-27T09:38:15.4678045Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 156, in __init__
2024-03-27T09:38:15.4679103Z     self.dist = self._prepare()
2024-03-27T09:38:15.4679491Z                 ^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4680548Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 222, in _prepare
2024-03-27T09:38:15.4681577Z     dist = self._prepare_distribution()
2024-03-27T09:38:15.4682025Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4683092Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 301, in _prepare_distribution
2024-03-27T09:38:15.4684316Z     return preparer.prepare_linked_requirement(self._ireq, parallel_builds=True)
2024-03-27T09:38:15.4684962Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4686220Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 525, in prepare_linked_requirement
2024-03-27T09:38:15.4687268Z     return self._prepare_linked_requirement(req, parallel_builds)
2024-03-27T09:38:15.4687774Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4688940Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 640, in _prepare_linked_requirement
2024-03-27T09:38:15.4689904Z     dist = _get_prepared_distribution(
2024-03-27T09:38:15.4690344Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4691459Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 71, in _get_prepared_distribution
2024-03-27T09:38:15.4692386Z     abstract_dist.prepare_distribution_metadata(
2024-03-27T09:38:15.4693765Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 54, in prepare_distribution_metadata
2024-03-27T09:38:15.4694855Z     self._install_build_reqs(finder)
2024-03-27T09:38:15.4695892Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 124, in _install_build_reqs
2024-03-27T09:38:15.4696789Z     build_reqs = self._get_build_requires_wheel()
2024-03-27T09:38:15.4697530Z                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4698615Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 101, in _get_build_requires_wheel
2024-03-27T09:38:15.4699590Z     return backend.get_requires_for_build_wheel()
2024-03-27T09:38:15.4700083Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4701111Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_internal/utils/misc.py", line 745, in get_requires_for_build_wheel
2024-03-27T09:38:15.4702250Z     return super().get_requires_for_build_wheel(config_settings=cs)
2024-03-27T09:38:15.4702890Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4704005Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 166, in get_requires_for_build_wheel
2024-03-27T09:38:15.4705071Z     return self._call_hook('get_requires_for_build_wheel', {
2024-03-27T09:38:15.4705687Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4706722Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 321, in _call_hook
2024-03-27T09:38:15.4707701Z     raise BackendUnavailable(data.get('traceback', ''))
2024-03-27T09:38:15.4708498Z pip._vendor.pyproject_hooks._impl.BackendUnavailable: Traceback (most recent call last):
2024-03-27T09:38:15.4709815Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 77, in _build_backend
2024-03-27T09:38:15.4710819Z     obj = import_module(mod_path)
2024-03-27T09:38:15.4711184Z           ^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4711937Z   File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/importlib/__init__.py", line 90, in import_module
2024-03-27T09:38:15.4712848Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-27T09:38:15.4713694Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-27T09:38:15.4714264Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2024-03-27T09:38:15.4715018Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2024-03-27T09:38:15.4715736Z   File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
2024-03-27T09:38:15.4716468Z   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2024-03-27T09:38:15.4717175Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2024-03-27T09:38:15.4717838Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2024-03-27T09:38:15.4718530Z   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
2024-03-27T09:38:15.4719301Z   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
2024-03-27T09:38:15.4719979Z   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
2024-03-27T09:38:15.4720661Z   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2024-03-27T09:38:15.4721800Z   File "/tmp/pip-build-env-95qtr52y/overlay/lib/python3.12/site-packages/setuptools/__init__.py", line 10, in <module>
2024-03-27T09:38:15.4722573Z     import distutils.core
2024-03-27T09:38:15.4723044Z ModuleNotFoundError: No module named 'distutils'
2024-03-27T09:38:15.4723438Z 
2024-03-27T09:38:15.9714259Z ##[error]Process completed with exit code 2.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,14 +1,27 @@
 # Release history
 
-## Version 0.9.5 [unpublished]
+## Version 0.9.6 [unpublished]
 
 ### Changes
 
--
+- 
 
 ### Bugfixes
 
--
+- 
+
+## Version 0.9.5
+
+### Changes
+
+- Updated package requirements for Brownie Bee user interface.
+
+### Bugfixes
+
+- Fix that categorical dimensions with more than two levels induces error when used 
+  together with SumEqual constraint.
+- Fix that Bokeh has changed naming convention related to sizes of circles in their
+  plots from "size" to "radius".
 
 ## Version 0.9.4
 

--- a/setup.py
+++ b/setup.py
@@ -39,15 +39,14 @@ setup(
     ],
     extras_require={
         "browniebee": [
-            "bokeh==3.1.1",
+            "bokeh==3.4.1",
             "deap==1.4.1",
-            "matplotlib==3.7.2",
-            "numpy==1.24.4",
+            "matplotlib==3.8.4",
+            "numpy==1.26.4",
             "pyYAML==6.0.1",
-            "scikit-learn==1.3.0",
-            "scipy==1.10.1",
+            "scikit-learn==1.4.2",
+            "scipy==1.13.0",
             "six==1.16.0",
-            "tornado==6.3.3",
         ]
     },
     long_description=long_description,

